Formula/percolator.rb

```diff
@@ -0,0 +1,41 @@
+class Percolator < Formula
+  # cite K_ll_2007: "https://doi.org/10.1038/nmeth1113"
+  desc "Semi-supervised learning for peptide identification from shotgun proteomics data"
+  homepage "http://percolator.ms"
+  url "https://github.com/percolator/percolator/archive/rel-3-05.tar.gz"
+  sha256 "5b746bdc0119a40f96bc090e02e27670f91eeb341736911750b170da7e5c06bb"
+  license all_of: ["Apache-2.0", "BSD-2-Clause", "BSD-3-Clause", "MIT"]
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "xerces-c" => :build
+  depends_on "xsd" => :build
+  depends_on "curl"
+  depends_on "icu4c"
+
+  on_macos do
+    depends_on "tokyo-cabinet"
+    depends_on "lbzip2"
+    depends_on "pbzip2"
+    depends_on "lzlib"
```

Comment:
Why are these dependencies needed only on macOS?

Reply:
I'm not sure -- I'm drawing from the [build script](https://github.com/percolator/percolator/blob/master/admin/builders/osx64_build.sh) found in the repo. (Which incidentally uses brew to install some deps, but not all of them)